prow/jobs/control-plane/control-plane-reconciler-upgrade-kyma2-latest-to-main-gardener.yaml

```diff
@@ -0,0 +1,67 @@
+# Code generated by rendertemplates. DO NOT EDIT.
+
+
+presubmits: # runs on PRs
+  kyma-project/control-plane:
+    - name: pre-main-control-plane-reconciler-upgrade-kyma2-latest-to-main-gardener
+      annotations:
+        pipeline.platform: "gardener_gcp"
+        pipeline.trigger: "pr-submit"
+      labels:
+        prow.k8s.io/pubsub.project: "sap-kyma-prow"
+        prow.k8s.io/pubsub.runID: "pre-main-control-plane-reconciler-upgrade-kyma2-latest-to-main"
+        prow.k8s.io/pubsub.topic: "prowjobs"
+        preset-bot-github-token: "true"
+        preset-cluster-version: "true"
+        preset-gardener-gcp-kyma-integration: "true"
+        preset-kyma-cli-stable: "true"
+      run_if_changed: '^resources/kcp/values.yaml|^resources/kcp/charts/mothership-reconciler/|^resources/kcp/charts/component-reconcilers/'
+      optional: true
```

Comment:
Should it be an optional job?

Reply:
As agreed with @k15r that we will keep this optional for now, until the reconciler becomes more stable.